Applications/Browser/main.cpp

```diff
@@ -106,6 +111,10 @@ int main(int argc, char** argv)
         window->set_title(String::format("%s - Browser", title.characters()));
     };
 
+    auto reload_action = GAction::create("Focus location box", { Mod_None, Key_F5 }, [&](auto&) {
```

Comment:
Yeah I realized that title mistake, fixed it in a later commit I pushed here too already ([275283f](https://github.com/SerenityOS/serenity/commit/275283fea063a8d682e3dc63d374fa214e2e15ae#diff-cea243a310d47ab948f7b5d8e7403c6eR114)), and I didn't know that :)
Would it still be ok to also have `F5` as Reload because imho that's a very common shortcut and used by all other browsers/file managers alike, but maybe implement it in, say, `HTMLView` instead?

Reply:
Dropped the shortcut entirely for now. It should probably be implemented globally when 2 shortcuts (e.g. `Ctrl` + `R` & also `F5`) can be added to [a single GAction like in make_reload_action()](https://github.com/SerenityOS/serenity/blob/master/Libraries/LibGUI/GAction.cpp#L79) :)